### PR TITLE
Reducers and action

### PR DESCRIPTION
**Pull request by dialite**
This pull request contains the following actions:

- Configure Redux store with reducers and actions for adding and removing books
- Use the ducks pattern in creating actions and reducers.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const ADD = 'bookstore/Book/ADD';
+const GET = 'bookstore/Book/GET';
+const REMOVE = 'bookstore/Book/REMOVE';
+
+export const addBook = (payload) => async (dispatch) => {
+  const book = { ...payload, item_id: payload.id, category: 'category' };
+  await apiAddBook(book);
+  dispatch({ type: ADD, book });
+};
+
```
Comment (HAMMAS-SALEEM) [inline src/redux/books/books.js]: Capture0

- [ ] You have a link of your license file in your readme file 😊 but it is broken 😨 Kindly fix this link to make your readme file descriptive and license file easily accessible to all viewers 👍

Comment (HAMMAS-SALEEM) [inline src/redux/books/books.js]: Capture

- [ ] It's great that you have a license file in your project but it looks like a template 🤔 Kindly add your **name**, **current year** and **app-type** 😊 to make your project licensed to you 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @dialite :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dialite** [comment]
Thank you. All required changes effected.

**Review by Davidosky007 — APPROVED** [review]
Hi @dialite 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Proper use of Gitflow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**